cmd/commands/datasetIngestor.go

```diff
@@ -147,8 +147,7 @@ For Windows you need instead to specify -user username:password on the command l
 		absFileListing := ""
 
 		if len(args) <= 0 || len(args) >= 3 {
-			log.Println("invalid number of args")
-			return
```

Comment:
was this changed to fatal because now datasetIngestor is called by cobra and the return does not stop execution? Should all return be changed then (I guess though cobra doesn't do anything more than calling this, so the return wouldn't hurt?)

Reply:
it was changed to fatal because it spares one line of code, and it's a fail condition anyways. I think it's the convention as well in Go, if we're not writing a library (in which case this would be a return WITH an error object and printing to stdout would be forbidden). In some of the other commands, this was already a Fatal( / ln / f) call.